ui.frontend/src/view/FormFieldBase.js

```diff
@@ -550,7 +554,9 @@ class FormFieldBase extends FormField {
                     if (!state.validationMessage) {
                         this.errorDiv.innerHTML = LanguageUtils.getTranslatedString(this.formContainer.getModel().lang, "defaultError");
                     }
-                } 
+                    this.errorDiv.setAttribute('tabindex', 0);
+                    this.errorDiv.setAttribute('role', 'complementary');
```

Comment:
It is usually incorrect to use role="complementary" on an error container. 
Use role="alert" or aria-live="assertive" for error messages, as this is the best practice(Optional)